package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1210,18 +1210,24 @@ def _donor2acceptor(self, donors, h_donors, acceptor):
                                            box=self.box,
                                            return_distances=True)
         if self.distance_type != 'heavy':
```

Comment:
Can we make this clearer:
```suggestion
        if self.distance_type == 'hydrogen':
```

Reply:
There's a comment in `__init__` "everything except 'heavy' will give the same behavior. This is not documented in the doc string which only lists "hydrogen" and "heavy". I'd like to make it so that only two documented options work. Anything else should give a ValueError.

If you don't want to add this to the PR please raise an issue.